controllers/etcdpeer_controller.go

```diff
@@ -176,8 +240,13 @@ func (r *EtcdPeerReconciler) Reconcile(req ctrl.Request) (ctrl.Result, error) {
 
 	log.V(2).Info("Found EtcdPeer", "name", peer.Name)
 
+	_, created, err := r.getOrCreatePvc(ctx, &peer)
```

Comment:
feels weird that we're not doing anything with the pvc, will we need it later? what for? is it worth propagating the volume's status through to the peer?

Reply:
I don't know yet. I'll  rename this function to `maybeCreatePvc` for now.